spec/plsql/schema_spec.rb

```diff
@@ -86,7 +86,7 @@
   it "should not connect with wrong port number" do
     expect {
       plsql.connect! @username, @password, :host => @host, :port => 9999, :database => @database
-    }.to raise_error(/no listener|could not establish the connection/)
+    }.to raise_error(/ORA-12541/)
```

Comment:
The test was failing if Oracle locale set to different than English.
I assume it's better to use Oracle error numbers than strings, as the strings are language-specific